srv/Odlc.srv

```diff
@@ -0,0 +1,4 @@
+Id mission
+string type
```

Comment:
Should make a `RequestType` (needs better name potentially) msg and put constants in there for `GET`, `PUT` etc. Then can make `int8` instead of string